### PR TITLE
Fix CTest to handle skipped case

### PR DESCRIPTION
**Pull request by ninjinkun**
According to our customer, CTest XML format don't have `<NamedMeasurement name="Execution Time" />` element in skipped tests. I fixed CTest plugin to handle missing that element. Also, I added a converter that handles a `notrun` case as a skipped case.

### CODE REVIEW

**Comment by ninjinkun** [comment]
I've tested this change with the customer's data. But I can't share it in this public repository.

**Comment by Konboi** [comment]
> I've tested this change with the customer's data. But I can't share it in this public repository.

I agree with you. 
But we should add a unit test for skip case and we can add a common skip test to a rocket car repo, can't you?

**Comment by ninjinkun** [comment]
No, I couldn't find the way to skip Google Test in the rocket-car-googletest.

**Comment by ninjinkun** [comment]
Okay, I'm going to add a test with anonymizing the customer's data.

File: launchable/test_runners/ctest.py [hunk 1]
```diff
@@ -52,32 +52,44 @@ def parse_func(p: str) -> ET.Element:
...
-                "./Results/NamedMeasurement[@name=\"Execution Time\"]/Value")
-            measurement = test.find("Results/Measurement/Value")
-            testcase = ET.SubElement(testsuite, "testcase", {
-                                     "name": test_name.text, "time": duration.text, "system-out": measurement.text})
+            if test_name != None:
+                duration = test.find(
+                    "./Results/NamedMeasurement[@name=\"Execution Time\"]/Value")
+                measurement = test.find("Results/Measurement/Value")
+                stdout = measurement.text if measurement != None else ''
+                duration = duration.text if duration != None else '0'
```
Comment (Konboi) [inline launchable/test_runners/ctest.py:64]: It's safe 👍

**Comment by ninjinkun** [comment]
@Konboi I modified an existing test case to skip case based on the customer's data.

**Review by Konboi — APPROVED** [review]
LGTM 🙆‍♀️

**Pull request merged**